src/main/java/io/github/opencubicchunks/cubicchunks/mixin/core/common/block/MixinPistonBlock.java

```diff
@@ -0,0 +1,18 @@
+package io.github.opencubicchunks.cubicchunks.mixin.core.common.block;
+
+import io.github.opencubicchunks.cubicchunks.CubicChunks;
+import net.minecraft.block.PistonBlock;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+
+@Mixin(PistonBlock.class)
+public class MixinPistonBlock {
+
+    @ModifyConstant(method = "canPush", constant = @Constant(ordinal = 0, expandZeroConditions = Constant.Condition.GREATER_THAN_OR_EQUAL_TO_ZERO))
+    private static int getCanPushGreater(int _0) { return -CubicChunks.worldMAXHeight; }
+
+    @ModifyConstant(method = "canPush", constant = @Constant(ordinal = 1))
```

Comment:
This does not target what you think it does, leads to `Narrowing conversion of <int> to <boolean>`

Reply:
I would recommend you read the javadoc on expandZeroConditions, as it states:
> However, explicitly specifying an <tt>intValue</tt> of <tt>0</tt> will cause this selector to also match explicit <tt>0</tt> constants in the method body as well.